unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/ChatController.cs

```diff
@@ -281,6 +267,20 @@ public List<ChatMessage> GetPrivateAllocatedEntriesByUser(string userId)
             .ToList();
     }
 
+    public void GetChannelInfo(string[] channelIds) => WebInterface.GetChannelInfo(channelIds);
+
+    public void GetChannelMembers(string channelId, int limit, int skip, string name) => WebInterface.GetChannelMembers(channelId, limit, skip, name);
+
+    public void GetChannelMembers(string channelId, int limit, int skip) => WebInterface.GetChannelMembers(channelId, limit, skip, string.Empty);
+
+    // called by kernel
+    [UsedImplicitly]
+    public void UpdateChannelMembers(string payload)
```

Comment:
can you please move the method to the group of where kernel messages are?

Reply:
Done!